rides/__init__.py

```diff
@@ -340,6 +341,11 @@ def join_ride(car_id, user):
             db.session.add(rider)
             db.session.add(car)
             db.session.commit()
+            u = User.query.get(username)
+            user_str = f"{u.firstname} {u.lastname}"
+            if not u[0].isdigit():
```

Comment:
Can you add a comment explaining what `u[0]` is and why you're checking this?

Reply:
Do we know that for certain? What do typical values look like? Is this how it normally checks?

Reply:
I checked the prod DB, all G Suite users are all digits and CSH usernames (linux usernames) cannot start with numbers